packages/react-components/react-avatar/stories/src/Avatar/AvatarSize.stories.tsx

```diff
@@ -23,7 +23,10 @@ export const Size = () => (
 Size.parameters = {
   docs: {
     description: {
-      story: 'An avatar supports a range of sizes from 16 to 128. The default is 32.',
+      story:
+        'An avatar supports a range of sizes from 16 to 128. The default is 32. \n\n' +
+        'Avoid using sizes 16 and 20 for interactive Avatars, ' +
```

Comment:
Should we add `px` units to these sizes?

```suggestion
        'An avatar supports a range of sizes from 16px to 128px. The default is 32px. \n\n' +
        'Avoid using sizes 16px and 20px for interactive Avatars, ' +
```

Reply:
ah, the reason I left them off (and I assume why there were left off in the original docs) is because the `size` prop on Avatar is unitless. So you author it as e.g. `<Avatar size={32}>`